src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -43,6 +43,10 @@ internal static void Init()
             //   * have high disk cost
 
             // We shouldn't create too many tasks.
+            Task.Run(() =>
+            {
+                var jsonDocument = System.Text.Json.Nodes.JsonObject.Parse("{\"test\":  \"test\"}");
```

Comment:
Is the purpose of this task just to load the assemblies?  If so, can you add a comment and you can just use the string `"1"` as valid json

Reply:
This is for early loading.
Comment is above.
"test" was replaced with "1".